package.json

```
@@ -314,7 +314,7 @@
     "electron-builder": "26.0.19",
     "eslint": "^9.36.0",
     "eslint-config-airbnb-typescript": "^18.0.0",
-    "eslint-config-expensify": "2.0.92",
+    "eslint-config-expensify": "^2.0.94",
```

We should not introduce a caret here:


```suggestion
    "eslint-config-expensify": "2.0.94",
```

as that would automatically bump the version on anything except a major version bump to 3.0